.github/workflows/peer-rec-CI.yml

```diff
@@ -0,0 +1,44 @@
+# This is a basic workflow to help you get started with Actions
+
+name: CI
+
+# Controls when the action will run. 
+on:
+  # Triggers the workflow on push or pull request events but only for the master branch
+  push:
+    branches: [ master, hosted-docs ]
+  pull_request:
```

Comment:
this is great! Lets have this run on every push to master. We don't need the hosted-docs branch necessarily and we don't want to update the live site until we get an approval. PRs going forward may have comment standards to help with auto documentation generation